cmd/server/cmd/server.go

```diff
@@ -408,17 +416,39 @@ func setupServer(sa *serverArgs, info map[string]template.Info, adapters []adptr
 		fatalf("Unable to listen on socket: %v", err)
 	}
 
+	sm := http.NewServeMux()
```

Comment:
Please keep existing `http.Handle` things and not defining servemux; actually this implicitly enables net/http/pprof (https://golang.org/pkg/net/http/pprof/).

Reply:
Ok fixed it.